internal/pkg/config/v1/config.go

```diff
@@ -73,17 +73,18 @@ var (
 type Config struct {
 	*config.BaseConfig
 
-	DisableUpdateCheck  bool                     `json:"disable_update_check"`
-	DisableUpdates      bool                     `json:"disable_updates"`
-	DisablePlugins      bool                     `json:"disable_plugins"`
-	DisableFeatureFlags bool                     `json:"disable_feature_flags"`
-	NoBrowser           bool                     `json:"no_browser" hcl:"no_browser"`
-	Platforms           map[string]*Platform     `json:"platforms,omitempty"`
-	Credentials         map[string]*Credential   `json:"credentials,omitempty"`
-	Contexts            map[string]*Context      `json:"contexts,omitempty"`
-	ContextStates       map[string]*ContextState `json:"context_states,omitempty"`
-	CurrentContext      string                   `json:"current_context"`
-	AnonymousId         string                   `json:"anonymous_id,omitempty"`
+	DisableUpdateCheck  bool                        `json:"disable_update_check"`
+	DisableUpdates      bool                        `json:"disable_updates"`
+	DisablePlugins      bool                        `json:"disable_plugins"`
+	DisableFeatureFlags bool                        `json:"disable_feature_flags"`
+	NoBrowser           bool                        `json:"no_browser" hcl:"no_browser"`
+	Platforms           map[string]*Platform        `json:"platforms,omitempty"`
+	Credentials         map[string]*Credential      `json:"credentials,omitempty"`
+	Contexts            map[string]*Context         `json:"contexts,omitempty"`
+	ContextStates       map[string]*ContextState    `json:"context_states,omitempty"`
+	CurrentContext      string                      `json:"current_context"`
+	AnonymousId         string                      `json:"anonymous_id,omitempty"`
+	SavedCredentials    map[string]*LoginCredential `json:"saved_credentials,omitempty"`
```

Comment:
@brianstrauch i chose to use a new map, as context states will be cleaned up when session timeout, and we can't auto-login back.